blog/2023-01-20-bind9-monitoring.md

```diff
@@ -0,0 +1,167 @@
+---
+slug: bind9-monitoring
+title: How to monitor and troubleshoot BIND 9
+description: Monitoring and troubleshooting BIND 9.
+authors: shyam
+tags: [how-to, monitor, troubleshoot, bind9, isc,  dns-server, domain-name-server, DNS-management, remote-control]
+keywords: [how-to, monitor, troubleshoot, bind9, isc,  dns-server, domain-name-server, DNS-management, remote-control]
+image: https://user-images.githubusercontent.com/24860547/213658626-e7e6d84a-11d9-43ff-aa95-17d2486a4cc3.png
+
+---
+
+Find out how to effectively and easily monitor and troubleshoot BIND 9 using Netdata
+
+<!--truncate-->
+## What is BIND 9?
+
+BIND 9 is a flexible, full-featured open source DNS system. 
+
+## Monitoring BIND 9 with Netdata
+
+The prerequisites for monitoring [BIND 9](https://www.isc.org/bind/) with Netdata are to have BIND and [Netdata installed](https://learn.netdata.cloud/docs/cloud/get-started) on your system. 
+
+Netdata auto discovers hundreds of services, and for those it doesn't turning on manual discovery is a one line configuration. For more information on configuring Netdata for BIND 9 monitoring please read the collector [documentation](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/bind).
+
+You should now see the `bind` section on the Overview tab in Netdata Cloud already populated with charts about all the metrics you care about.
+
+Netdata has a public [demo space](https://app.netdata.cloud/spaces/netdata-demo) (no login required) where you can explore different monitoring use-cases and get a feel for Netdata.
+
+## What BIND 9 metrics are important to monitor?
+
+### Clients
+
+**Recursive Clients**
```

Comment:
Is this description correct? Aren't these the clients making recursive queries?

Reply:
updated